project/Doc.scala

```diff
@@ -75,7 +78,7 @@ object UnidocRoot extends AutoPlugin {
       |  }
       |</script>""".stripMargin.replaceAll("\n", "\\\\n").replaceAll("\"", "\\\\\"")
 
-  override lazy val projectSettings = javaUnidocSettings ++ Seq(
+  override lazy val projectSettings = scalaJavaUnidocSettings ++ Seq(
     unidocAllSources in (JavaUnidoc, unidoc) ++= allGenjavadocSources.value,
     unidocAllSources in (JavaUnidoc, unidoc) := {
```

Comment:
Yep, not sure how to do that for Scala, will look sometime later once the Scala APIs are finished.